test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -752,6 +757,30 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 		framework.ExpectNoError(framework.WaitForReadyNodes(c, len(nodes.Items), nodesRecoverTimeout))
 	})
 
+	It("should add new node and new node pool on too big pod [Feature:ClusterSizeAutoscalingScaleWithNAP]", func() {
```

Comment:
Can you add ```framework.SkipUnlessProviderIs("gke")```? Let's not rely on selectors in current test-infra jobs for this.

Reply:
Done